### ISSUE TITLE
Disable mixing hashmap + array notation

### ISSUE THREAD

**Issue by apfelbox** [kalessil/phpinspectionsea#1493]
Description (screenshot):
--

Well, I basically wrote this bug and it took some time to find it:

![2020-01-11 at 13 04](https://user-images.githubusercontent.com/1032411/72543775-81a3f300-3886-11ea-8a49-9a70cae46a70.png)

I am not sure how often people mix an indexed array with a hashmap, but maybe it could be an option to show a warning?

Example
--

Is this code style often used in PHP?

```php
return [
    "key" => $value,
    $other,
    $someMore,
    "otherKey" => $otherValue,
    $end,
];
```

If not -> maybe show a warning if these two styles are mixed?

**Comment by kalessil** [comment]
Thank you for sharing @apfelbox !

**Comment by kalessil** [comment]
Implemented!

**Comment by orklah** [comment]
Hi,

I didn't find this inspection on the inspection panel with this name and even though I activated almost all inspection, it doesn't seem to flag those cases.

Maybe broken?
